src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1039,7 +1039,12 @@ bool AdaptDispatch::ResetPrivateModes(const std::basic_string_view<DispatchTypes
 // - True if handled successfully. False otherwise.
 bool AdaptDispatch::SetKeypadMode(const bool fApplicationMode)
 {
-    return _pConApi->PrivateSetKeypadMode(fApplicationMode);
+    // TODO GH#XXXX:
+    //   This is a temporary replacement to enable passhthrough
+    //   mode for Windows Terminal. Replace with proper _pConApi
+    //   call below when ConPty has been properly updated.
+    _pConApi->PrivateSetKeypadMode(fApplicationMode);
+    return false;
```

Comment:
only return false if we're in `IsConsolePty` mode (like other things around this file)